micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/Login-v2/signUp.js

```
@@ -0,0 +1,176 @@
+          const url = buildOtpUrl(window?.contextPath, tenantId);
+          window.location.replace(url);
+        },
+      }
+    );
+  };
+
+  const closeToast = () => {
+    setShowToast(null);
+  };
+
+  let config = [{ body: propsConfig?.inputs }];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'config' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)